README.md

```diff
@@ -19,6 +19,37 @@ used to run the batch on your behalf. There's a dedicated Batch service
 account for each project (e.g. "tob-wgs"), which helps with bucket permission
 management and billing budgets.
 
+## Access through CLI
+
+If you have the `analysis-runner` CLI installed, you can request the analysis-runner
```

Comment:
Can we add a link here to the installation instructions? (E.g. in the getting started guide.)

Reply:
Yep, I've added:

> Please see the [Team Docs](https://github.com/populationgenomics/team-docs/) for
installation instructions.

And when it's deployed (and I can double check the actual install instructions), I'll add them to the team docs. I've avoided linking to a specific page in case links move.